pkg/nmi/server/server.go

```diff
@@ -104,8 +105,8 @@ func (s *Server) Run() error {
 	}
 	rtr.PathPrefix("/").HandlerFunc(s.defaultPathHandler)
 
-	klog.Infof("listening on port %s", s.NMIPort)
-	if err := http.ListenAndServe("localhost:"+s.NMIPort, rtr); err != nil {
+	klog.Infof("listening on %s:%s", s.NMIHost, s.NMIPort)
+	if err := http.ListenAndServe(fmt.Sprintf("%s:%s", s.NMIHost, s.NMIPort), rtr); err != nil {
```

Comment:
The iptable rules need to use the host IP that NMI is listening on, so this needs to be updated too:
https://github.com/Azure/aad-pod-identity/blob/6cf9bac500d8f3f29492a6fd53f2e8990d51d9f0/pkg/nmi/server/server.go#L117-L119

Reply:
Uhm good point. Lemme think about that, it might be a little tricky